File: src/me/krickl/memebotj/InternalCommands/ChannelInfoCommand.java
package me.krickl.memebotj.InternalCommands;

import me.krickl.memebotj.ChannelHandler;
import me.krickl.memebotj.CommandHandler;
import me.krickl.memebotj.UserHandler;

public class ChannelInfoCommand extends CommandHandler {
	public ChannelInfoCommand(String channel, String command, String dbprefix) {
		super(channel, command, dbprefix);
		this.setAccess("moderators");
		this.setNeededCommandPower(25);
	}

	@Override
	protected void commandScript(UserHandler sender, ChannelHandler channelHandler, String[] data) {
		channelHandler.sendMessage(String.format("Is live: %b || Points per update %f", channelHandler.isLive(), channelHandler.getPointsPerUpdate()), this.getChannelOrigin());
	}
}

File: src/me/krickl/memebotj/InternalCommands/ChannelInfoCommand.scala
package me.krickl.memebotj.InternalCommands

import me.krickl.memebotj.ChannelHandler
import me.krickl.memebotj.CommandHandler
import me.krickl.memebotj.UserHandler

class ChannelInfoCommand(channel: String, command: String, dbprefix: String)
		extends CommandHandler(channel, command, dbprefix) {

	this.setAccess("moderators")

	this.setNeededCommandPower(25)

	override def commandScript(sender: UserHandler, channelHandler: ChannelHandler, data: Array[String]) {
		channelHandler.sendMessage(f"Is live: ${channelHandler.isLive} || Points per update ${channelHandler.getPointsPerUpdate} || purge regex: ${channelHandler.getUrlRegex}", this.getChannelOrigin)
	}
}
